src/js/services/WebsiteService.ts

```diff
@@ -54,6 +55,13 @@ class WebsiteService {
         texts: batch.map(item => {
           // This is temporary solution
           let isSEO = false
+          let elementSeoItem = TranslationElementCandidates.get(item.tagName)
+
+          if (elementSeoItem && elementSeoItem.type === TranslatableItemType.ELEMENT_SEO) {
```

Comment:
Kāpēc vajag papildus pārbaudi, nepietiek ar to kas zemāk (uz item.type)?

Reply:
TranslationElementCandidates faktiski ir saraksts ar tagiem, kurus vēlamies saglabāt kā SEO elementus, lai translation hub var filtrēt viņus pēc tam pēc SEO filtra. lai neiejaukties esošajā SEO attributes loģikā, te vienkārši pirms nosūtīšanas tulkošanai papildus item check, vai item tags ir iekļauts TranslationElementCandidates, ja ir, tad tas ir seo elements

Reply:
Nesapratu, kāpēc neiejaukties esošajā SEO loģikā, neuzstādīt title kā SEO elementu? heading tagus mums arī vajag pie SEO rādīt?

Reply:
jo esošajā loģikā nemaz nav saprotams, kur jēdzīgi to tipu nomainīt un kāpēc eksistēja TranslationElementCandidates, ja reāli TITLE un visi H1-H6 tika sūtīti kā parasti elementi, nevis seo? 
par heading tagiem nezinu, no Bertas jāprecizē

Reply:
Varbūt @gunpuz var pakomentēt, kā tas strādā? Heading tagus manuprāt labāk nelikt kā SEO, jo tie ir redzami lapā.

Reply:
Katrā ziņā, labāk papētīt sīkāk, lai pēc tam nekonfliktē tās loģikas

Reply:
ir viena ideja, kur to check uz tagiem no TranslationElementCandidates var ielikt DOMTranslation, bet būs jāpārbauda

Reply:
> Varbūt @gunpuz var pakomentēt, kā tas strādā? Heading tagus manuprāt labāk nelikt kā SEO, jo tie ir redzami lapā.

īstenībā, laikam jau heading tagi arī ir būtiski seo
https://www.link-assistant.com/news/html-tags-for-seo.html#:~:text=HTML%20SEO%20tags%20are%20bits,pages%20appear%20in%20search%20results.

Reply:
Ja var rādīt abās kategorijās viņus, tad var likt kā seo. Bet ja lietotājs rediģē redzamos elementus, un tur neparādās nosaukumi, tāpēc ka tie ir svarīgi arī seo, tad labāk atstāt kā parastā tipa elementus

Reply:
tu domā saglabāt vienu elementu gan kā seo, gan parastu?

Reply:
Nē, domāju, varbūt var frontendā noteikt, ka elements pieder gan seo, gan parastajiem. Bet laikam nav tā paredzēts

Reply:
Heading elementi arī ir SEO elementi, nav nepieciešams prasīt Bertai. 

Ja Te tips ir nodefinēts kā SEO, tad droši vien tas ir speciāli un apakšā tam ir iemesls.
![image](https://github.com/user-attachments/assets/2418684d-5bbf-442f-9753-213b066d229d)

Tikai kāpēc vajag papildus vēl atribūtu nav skaidrs:
![image](https://github.com/user-attachments/assets/c46eae51-142f-4246-bbbe-b5a18cb8e01c)

Key jau paska atribūta nosaukumu. Kāpēc vajag vēl vienu atslēgu kas dublē atribūta nosaukumu?

Reply:
taisnība, izņēmu attributeName, tā vietā var izmantot key, kas ir arī tag nosaukums